### ISSUE TITLE
Asymmetric Donchian Channel

### ISSUE THREAD

**Issue by adriantorrie** [QuantConnect/Lean#150]
This is an extension to #47.

Currently `DonchainChannel` takes one argument, `period`, and uses this to determine both the upper and lower limits of the channel.

```csharp
public DonchianChannel(string name, int period)
            : base(name)
{
    UpperBand = new Maximum(name + "_UpperBand", period);
    LowerBand = new Minimum(name + "_LowerBand", period);
}
```

An asymmetric channel would allow different periods to be set for the upper channel and lower channels separately, perhaps like this?

```csharp
public DonchianChannel(string name, int upperPeriod, int lowerPeriod)
            : base(name)
{
    UpperBand = new Maximum(name + "_UpperBand", upperPeriod);
    LowerBand = new Minimum(name + "_LowerBand", lowerPeriod);
}

public DonchianChannel(string name, int period)
            : this(name, period, period)
{
}
```

**Comment by mchandschuh** [comment]
Closed in PR #151

**Comment by mandakamitramedia** [comment]
adriantorrie,
may share in form of xml file, txs
